src/ofdm_demod.c

```diff
@@ -83,7 +83,7 @@ void opt_help() {
     fprintf(stderr, "  --ns           Nframes   Number of Symbol Frames (8 default)\n");
     fprintf(stderr, "  --tcp            Nsecs   Cyclic Prefix Duration (.002 default)\n");
     fprintf(stderr, "  --ts             Nsecs   Symbol Duration (.018 default)\n");
-    fprintf(stderr, "  --bandwidth    [0|1|2]   Select Auto Phase Estimate (0) Low (1) High (2) RX mode (default 2)\n");
+    fprintf(stderr, "  --bandwidth    [0|1|2]   Select Lock High Phase Estimate (0) Low (1) High (2) RX mode (default 2)\n");
     fprintf(stderr, "  --interleave     depth   Interleaver for LDPC frames, e.g. 1,2,4,8,16 (default is 1)\n");
```

Comment:
Maybe ```--phase_est_mode 0|1```, hmm that's a long name but ```bandwidth``` is a bit general so I'm up for other suggestions!